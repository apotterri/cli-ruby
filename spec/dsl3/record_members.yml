- !group &developers developers

- !user
  id: alice
  members: &foobar
  - *developers
  -
    role: !group ops
    admin: true
