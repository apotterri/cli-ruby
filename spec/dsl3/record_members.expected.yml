---
- &1 !group
  id: developers
- !user
  id: alice
  members:
  - !member
    role: *1
  - !member
    role: !group
      id: ops
    admin: true
